crates/ruff_workspace/src/options.rs

```diff
@@ -1415,6 +1415,15 @@ impl<'de> Deserialize<'de> for Alias {
         D: Deserializer<'de>,
     {
         let name = String::deserialize(deserializer)?;
+        // Assignments to these names are SyntaxErrors
+        // see the note here:
+        // https://docs.python.org/3/library/constants.html#debug__
+        if matches!(&*name, "__debug__" | "True" | "False" | "None") {
```

Comment:
I think we can skip "True", "False" and "None" here as they are already being checked by the `is_identifier` check below:

https://github.com/astral-sh/ruff/blob/5137fcc9c81610f687b6cb45413ef83c2c5eea73/crates/ruff_python_stdlib/src/identifiers.rs#L19-L22

Reply:
Nice catch! 

I guess that's a somewhat confusing function name because keywords _are_ identifiers, see: https://docs.python.org/3/reference/lexical_analysis.html#keywords

and also:

```console
>>> "None".isidentifier()
True
```

Reply:
Yes, keywords are identifiers but they cannot be used as an identifier. I think the function is used to test the latter.